app/views/dashboard/index.html.haml

```diff
@@ -1,17 +1,18 @@
 - content_for :after_title, " - " + t("application.my_dashboard")
 
-#react_root
+#react_root{data: {default_course_type: ENV["default_course_type"],
+                   course_string_prefix: ENV["default_course_type"].constantize.new.string_prefix}}
```

Comment:
This breaks if default_course_type is not set. Let's do this in the presenter.

I'm also thinking that it makes sense to make string_prefix a class method instead of an instance method, so that we can do things like this without creating an instance just to get a hard-coded string out of the course type.